README.md

```diff
@@ -1,39 +1,108 @@
 - node 10+
 - aws cli
 - tsc
+- jq
 
-### Getting Started with Full Deploy
+To deploy using this approach, you must first set several values inside the `package.json` file in the `source` folder.
 
-Prerequisites:
+- Set your deployment region in the `stack->region` property, replacing `"%%REGION%%"`. Unlike the regular CICD deploy, this approach will not pull the AWS region from your current AWS profile.
+- Enter your email into the `email` property, replacing `"%%USER_EMAIL%%"`
 
-- Set your deployment region in the stack->region property of package.json. Make sure this also matches the region you have set via aws configure
-- If you have never used CDK before, then the deployment command below may fail with a message saying that you first need to run `cdk bootstrap {accountId}/{region}`. This will deploy a small stack with resources for running CDK. Afterwards, run the below yarn command again.
+Now switch to the source directory, and use yarn to deploy the solution:
+
+```
+cd ./source
+```
 
 ```
 yarn && yarn deploy
 ```
 
 The cli will prompt for approval on IAM Roles and Permissions twice in the full deploy. Once for the backend stack and then again for the client stack. The cli will prompt for an email. After the deploy is complete, an email will be sent to address provided with credentials for logging in.
 
+Note:
+
+This will create 3 or 4 S3 buckets that will have to be manually deleted when the stack is destroyed (Cloudformation does not delete them, in order to avoid data loss).
+
+- 2 for documents (sample and general documents)
+- 1 for the client bucket
+- 1 for CDK toolkit (if this is your first time using CDK)
+
 ### Development Deploy Commands
 
-- `yarn deploy:stack` : deploys or updates the backend stack
+- `yarn deploy:backend` : deploys or updates the backend stack
 - `yarn deploy:client` : deploys or updates the client app
 - `yarn deploy:setup-samples` : push sample docs to s3
 - `yarn deploy:setup-user` : initiated prompts to set up a user
 - `yarn deploy:show` : displays the url of the client app
-- `yarn destroy` : tears down the CloudFormation backend stack
+- `yarn destroy` : tears down the CloudFormation backend and client stacks
 
 ### Development Deploy Workflow and stacknaming
 
-The `package.json` script node `stackname` sets the stackname for the deploy commands. Throughout development it has been imparative to maintain multiple stacks in order to allow client app development and stack architecture development to work without creating breaking changes. When a new stackname is merged into develop it should have the most up to date deployments.
+The `package.json` script node `stackname` sets the stackname for the deploy commands. Throughout development it has been imperative to maintain multiple stacks in order to allow client app development and stack architecture development to work without creating breaking changes. When a new stackname is merged into develop it should have the most up to date deployments.
```

Comment:
I know this can be out of the scope of this PR, but have there been any thoughts of using Nested stacks?
We have one master stack that can then spin both the DUS and Client stack for Yarn deploy as well and one yarn destroy.